### PR TITLE
Add iburst ntp option

### PR DESCRIPTION
**Pull request by imobachgs**
Add the iburst option. I didn't introduce this option on the initial version because an (already solved) bug in CFA. Now we can happily add it as we do in openSUSE.

### CODE REVIEW

File: test/lib/y2system_role_handlers/dashboard_role_finish_test.rb [hunk 1]
```diff
@@ -43,6 +43,7 @@
         handler.run
         records = ntp_conf.records.select { |r| r.type == "server" }
         expect(records.map(&:value)).to eq([ntp_server])
+        expect(records.first.options).to eq(["iburst"])
```
Comment (lslezak) [inline test/lib/y2system_role_handlers/dashboard_role_finish_test.rb]: Um, I find it a bit inconsistent, on the previous lines you check *all* record values, but here you check only the first item.

Why not using something similar here? E.g. `expect(records.map(&:options)).to eq([["iburst"]])`

File: src/lib/y2system_role_handlers/dashboard_role_finish.rb [hunk 2]
```diff
@@ -70,9 +70,12 @@ def update_ntp_conf
       ntp_conf.save
     end
 
+    SERVER_OPTIONS = ["iburst"].freeze
+    # Build a server record
```
Comment (lslezak) [inline src/lib/y2system_role_handlers/dashboard_role_finish.rb]: Missing an empty line as a separator between the constant and the method?

**Review by lslezak — COMMENTED** [review]
Test consistency?

**Review by lslezak — APPROVED** [review]
LGTM

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**